### ISSUE TITLE
Kalman filters

### ISSUE THREAD

**Issue by yakir12** [joshday/OnlineStats.jl#245]
[Feature request] Wouldn't it be a perfect fit and make total sense to have a Kalman filter as one of the stats here?

**Comment by joshday** [comment]
Yes, either that or to have an existing Kalman filter package depend on OnlineStatsBase.  

I'm interested in adding it here, but I probably won't have the bandwidth to work on it anytime soon.

**Comment by guilhermebodin** [comment]
I could help in this case, we have a fairly performant Kalman filter implemented here https://github.com/LAMPSPUC/StateSpaceModels.jl/blob/master/src/filters/univariate_kalman_filter.jl

Currently, if we have all the matrices information (we don't need to estimate any parameters) the filter states are already calculated in an "online" manner.
